File: iraf/unix/boot/spp/rpp/rppfor/allblk.f
      integer function allblk (buf)
      integer buf (100)
      integer i
      allblk = 1
      i = 1
23000 if (.not.(buf (i) .ne. 10 .and. buf (i) .ne. -2))goto 23002
      if (.not.(buf (i) .ne. 32))goto 23003
      allblk = 0
      goto 23002
23003 continue
23001 i = i + 1
      goto 23000
23002 continue
      return
      end

File: unix/boot/spp/rpp/rpprat/allblk.r
#-h-  allblk			  486  local   12/01/80  15:53:44
# allblk - determine if line consists of all blanks
   include  defs

# this routine is called by outdon, and is here to fix
# a bug which sometimes occurs if two or more includes precede the
# first line of executable code.  Could not trace down the cause

   integer function allblk (buf)
   character buf (ARB)

   integer i

   allblk = YES
   for (i = 1; buf (i) != NEWLINE & buf (i) != EOS; i = i + 1)
      if (buf (i) != BLANK) {
	 allblk = NO
	 break
	 }

   return
   end
